signal/train_game.py

```diff
@@ -326,7 +324,8 @@ def baseline(args):
     model_name = train_helper.get_filename_from_baseline_params(args)
     run_folder = file_helper.get_run_folder(args.folder, model_name)
 
-    metrics_helper = MetricsHelper(run_folder, args.seed)
+    logger = Logger(run_folder, print_logs=(not args.disable_print))
```

Comment:
I think that in general we shouldn't use the print facility. But that's probably something we should address in a different issue/pull request and outside the scope of this batch of commits.